packages/vm/lib/evm/opcodes.ts

```diff
@@ -284,13 +280,22 @@ function getFullname(code: number, name: string): string {
  * @returns {OpcodeList} Opcodes dictionary object.
  */
 export function getOpcodesForHF(common: Common): OpcodeList {
-  let opcodeBuilder = { ...opcodes }
+  let opcodeBuilder: any = { ...opcodes }
 
   for (let fork = 0; fork < hardforkOpcodes.length; fork++) {
     if (common.gteHardfork(hardforkOpcodes[fork].hardforkName)) {
       opcodeBuilder = { ...opcodeBuilder, ...hardforkOpcodes[fork].opcodes }
     }
   }
 
-  return opcodeBuilder
+  for (let key in opcodeBuilder) {
+    let baseFee = common.param('gasPrices', opcodeBuilder[key].name.toLowerCase())
```

Comment:
Post-review note: this `toLowerCase()` is a pretty dangerous assumption - since `Common` is using camelCase for multi-word parameter names - I actually wonder that this is working at all on second thought since `Common` throws on using the wrong case. Probably some lucky coincidence on what values are read in this context. 😄 

We should definitely improve here otherwise this is bound to go wrong at some PIT. Not sure how to tackle:

- Ignore case on the the common side respectively only compare lower case versions? Side effects?
- Use the upcoming release on Common and move everything to lowercase and enforce? Hmm.
- Other suggestions?

Not completely convinced myself on any of the solutions proposed yet.

Reply:
I created a quick script to extract opcode fees from the opcodes object and then lower cased them since that seemed to be the standard in Common (had sload gasprice instead of sLoad or something different). But I agree we should standardize this or something, good point.

Reply:
The thing is, lower case is NOT standard in `Common` but camelCase and camelCase names are all over the place - see e.g. the [chainstart.json](https://github.com/ethereumjs/ethereumjs-vm/blob/master/packages/common/src/hardforks/chainstart.json) file. I would guess that this didn't trigger an exception here is out of pure randomness, maybe all the camelCase names are for non-base fee gas prices or precompiles (like `ecAdd` e.g.), but that was likely pure luck here. 😄 

Therefore the somewhat urgency of the comment. Or are we talking past each other somehow?

Reply:
I understand your concern. I think we can change it to camelCase just to be consistent, but to implement this in a clean way in `opcodes.ts` we also need to change all uppercase opcodes there to the right camelCased opcodes.